src/cli/cli_app.js

```diff
@@ -396,11 +405,33 @@ class CLI {
     /** init Ethereum API
      * */
     if (this._initEthereum) {
+      let enigmaContractAbi = null;
+      let accountKey = this._ethereumKey;
+      if (this._enigmaContractAbiPath) {
+        try {
+          let raw = await utils.readFile(this._enigmaContractAbiPath);
+          enigmaContractAbi = JSON.parse(raw).abi;
+        }
+        catch(e) {
+          console.log(`Error in reading enigma contract API ${this._enigmaContractAbiPath}`);
+          return;
+        }
+      }
+      if (this._ethereumKeyPath) {
+        try {
+          accountKey = await utils.readFile(this._ethereumKeyPath);
+        }
+        catch(e) {
+          console.log(`Error in reading account key ${this._ethereumKeyPath}`);
+          return;
+        }
+      }
       builder.setEthereumConfig({
-        ethereumUrlProvider: this._ethereumWebsocketProvider,
+        urlProvider: this._ethereumWebsocketProvider,
         enigmaContractAddress: this._enigmaContractAddress,
-        ethereumAddress: this._ethereumAddress,
-        ethereumContractAbiPath: this._enigmaContractAbiPath
+        accountAddress: this._ethereumAddress,
+        enigmaContractAbi: enigmaContractAbi,
```

Comment:
Can use ES6 syntax and just use:
`enigmaContractAbi,`
`accountKey`

Reply:
Done